bundle/Core/SiteAccess/Processor.php

```diff
@@ -104,7 +104,13 @@ function (UserGroup $group) {
             $data['groupIds'] = null;
         }
 
-        $content = $this->twig->render('@ezdesign/site_configuration/siteaccess.yaml.twig', $data);
+        // Check type content type (novaezsiteaccessfactory_home_page_vide or novaezsiteaccessfactory_home_page)
+        if ($configuration->getModele() == 1) {
```

Comment:
```suggestion
        if ($configuration->getModele() === true) {
```

Reply:
At the second time, we will have other options in the select